src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -428,12 +420,7 @@ private static void addScmDetails(Run run, TaskListener listener, List<Fact> fac
                 for (Object o : changeSet.getItems()) {
                     ChangeLogSet.Entry entry = (ChangeLogSet.Entry) o;
                     entries.add(entry);
-                    try {
-                        files.addAll(entry.getAffectedFiles());
-                    } catch (Throwable e) {
-                        listener.getLogger().println(e.getMessage());
-                    }
-
+                    files.addAll(entry.getAffectedFiles());
```

Comment:
API says there is no need to catch as there is no exception

Reply:
@damianszczepanik  You sure about this. As I remember, the getAffectedFiles() used to throw some exception (Not sure, have forgot) and hence we were not sending notifications. So, I thought of catching the exception and ignore it. Even if we encounter any exception, only that field would be missing and all other fields would be present in the card.

Reply:
@damianszczepanik  You sure about this. As I remember, the getAffectedFiles() used to throw some exception (Not sure, have forgot) and hence we were not sending notifications. So, I thought of catching the exception and ignore it. Even if we encounter any exception, only that field would be missing and all other fields would be present in the card.

Reply:
Yes pretty sure:
Jenkins:
https://github.com/jenkinsci/jenkins/blob/08def67a18eee51de9f3f99bc2a792fee1c160e0/core/src/main/java/hudson/scm/ChangeLogSet.java#L235

https://github.com/jenkinsci/git-plugin/blob/ce572609afc6738472b4dba3769e8b950204c2fd/src/main/java/hudson/plugins/git/GitChangeSet.java#L347
https://github.com/jenkinsci/perforce-plugin/blob/4beaabe78faf62a8cfad340e933f1ff0bd2d07c1/src/main/java/hudson/plugins/perforce/PerforceChangeLogEntry.java#L39
https://github.com/jenkinsci/subversion-plugin/blob/8ef22113a307da105a5d54d7db7ce7b56af85818/src/main/java/hudson/scm/SubversionChangeLogSet.java#L349
https://github.com/jenkinsci/cvs-plugin/blob/9390d5ae0b07c29c87adc87ae10be302507a5347/src/main/java/hudson/scm/CVSChangeLogSet.java#L417